content/md/en/docs/maintain/monitor.md

```diff
@@ -4,59 +4,174 @@ description:
 keywords:
 ---
 
-Monitoring network operations and performance is an important part of maintaining a healthy, stable, and secure parachain ecosystem.
+Monitoring network operations and performance is an important part of maintaining a healthy, stable, and secure ecosystem.
 By default, Substrate nodes provide low-level **telemetry** components to automatically collect and transmit detailed information about the operation of each node in the network.
 Telemetry is the underlying mechanism—sensors, probes, or instrumentation—for collecting and transmitting data points as they occur to an endpoint so that you can observe, monitor, and alert on system behavior.
+
 As a developer or node operator, there's rarely any need to know the implementation details of how these low-level telemetry components provide information about computer and network operations.
-At a high level, the information collected from each node is sent to a default telemetry server and aggregated for display in a front-end dashboard.
+At a high level, the information collected from each public node is sent to a default telemetry server and aggregated for display in a front-end dashboard.
+
+The following diagram provides a simplified overview of the workflow.
 
 ![Default telemetry for Substrate nodes](/media/images/docs/telemetry-overview.png)
 
-## Changing the telemetry server
+## View telemetry data
+
+The [Polkadot telemetry](https://telemetry.polkadot.io/) dashboard provides a real-time view of how currently online nodes are performing.
+You can choose the information you want to display by turning visible columns on and off from the list of columns available.
+
+To see the list of columns and modify which columns are displayed, click **Settings**.
+
+![Click Settings to modify the visible columns](/media/images/docs/list-view-control.png)
+
+The columns you can toggle on and off include:
+
+- **Validator** to display an indicator to differentiate validator nodes from other nodes. 
```

Comment:
Just as a note; this column I think hasn't been working properly for a whlie (owing to the data it expects not being provided now) so it may be fixed or removed. Probably not an issue here though :)